lib/shopify_cli/tunnel.rb

```diff
@@ -178,6 +178,14 @@ def install(ctx)
 
     def fetch_url(ctx, log_path)
       LogParser.new(log_path)
+    rescue NgrokError => e
+      case e.message
+      when /\AYour account is limited to 1 simultaneous ngrok client session/
```

Comment:
It'd be awesome if Ngrok used status codes when exiting their process unsuccessfully. I created [this issue](https://github.com/Shopify/shopify-cli-planning/issues/84) to keep track of these things to share feedback with them.

Reply:
@pepicrft I think they actually do use [error codes](https://ngrok.com/docs/errors). Sounds like I should edit this to look for the error code rather than the text, and maybe link to that doc in a comment here.